pkg/apis/internal.admin.acorn.io/v1/types.go

```diff
@@ -15,6 +15,27 @@ func (in *ClusterComputeClassInstance) NamespaceScoped() bool {
 	return false
 }
 
+func (in *ClusterComputeClassInstance) ForRegion(region string) bool {
+	for _, r := range in.SupportedRegions {
+		if r == region {
+			return true
+		}
+	}
+	in.SupportedRegions = append(in.SupportedRegions, region)
+	return true
+}
+
+// ForOtherRegions returns true if there are other regions that this instance is supported in.
+func (in *ClusterComputeClassInstance) ForOtherRegions(region string) bool {
+	for i, r := range in.SupportedRegions {
+		if r == region {
+			in.SupportedRegions = append(in.SupportedRegions[:i], in.SupportedRegions[i+1:]...)
```

Comment:
It looks like this is removing the matching region from the list. I'm a bit confused why a call to this function would remove a supported region from a ClusterComputeClassInstance.